README.md

```diff
@@ -14,12 +14,15 @@ If you would like to collaborate, please email Andreu Font-Ribera (afont@ifae.es
 - Create a new conda environment. It is usually better to follow python version one or two behind. In January 2024, the latest is 3.12, so we recommend 3.11.
 
 ```
-conda create -n cup1d -c conda-forge python=3.11 camb mpich mpi4py fdasrsf pip=24.0
+conda create -n cup1d -c conda-forge python=3.11
```

Comment:
Please check out my comment for camb, fdasrsf and pip in lace. Regarding mpich and mpi4py, we need to install these via conda so the mpi compiler is installed in clusters without it. This is extremely important, otherwise the installation will not work

Reply:
A cluster without MPI support is something... Supporting every possible architecture will be difficult for you. NERSC and your laptop are enough in my opinion.

Reply:
I agree that in the long term (or hopefully intermediate term) we should focus on running at NERSC and similar clusters to make our life easier. However, right now Nils is running his MontePython runs at a local cluster, and this short-term solution was the only way for him to use the LaCE-Nyx emulator (as far as I remember)